CDCgov/prime-reportstream
# Update tests to unmockk objects to fix inconsistent test failures

This PR adjusts two unit tests that mockk objects to unmockk them at the conclusion of the test in order to resolve an inconsistent failure in other unit tests that use those objects.

Test Steps:
1. Run Gradle tasks `clean` and `package`.
2. Confirm `package` is successful (unit tests are run in the process; errors were previously observed in this process on M2 Mac environments)

## Changes
- Adjust two unit tests to unmockk objects at the end.

## Checklist

### Testing

## Linked Issues
Addresses a change made in #9233.
